velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -447,7 +475,13 @@ void AggregationFuzzerBase::testPlan(
       abandonPartial,
       maxDrivers);
 
-  // Compare results or exceptions (if any). Fail is anything is different.
+  if (FLAGS_enable_oom_injection) {
```

Comment:
We can check the result if both expected and actual don't have exceptions? Thanks!